deployments/dev-r/images/default/infra-requirements.txt

```diff
@@ -10,10 +10,10 @@
 # FIXME: Freeze this to get exact versions of all dependencies
 notebook==7.0.7
 jupyterlab==4.0.11
-nbgitpuller==1.2.1
+nbgitpuller==1.1.1
 jupyter-resource-usage==1.0.1
 # Matches version in images/hub/Dockerfile
-jupyterhub==4.1.5
+jupyterhub==4.0.1
```

Comment:
change this to `4.0.2`, which matches the version on the dev-r staging hub.

Reply:
done